tests/__snapshots__/test_samples.ambr

```diff
@@ -0,0 +1,214 @@
+# serializer version: 1
```

Comment:
syrupy appears to support: `SingleFileSnapshotExtension`. The ambr is the default, but having one file per test might be more aligned with our usual btest setup.

Reply:
Nice, adjusted.